src/content/docs/it/tutorial/1-setup/3.mdx

```diff
@@ -0,0 +1,71 @@
+---
+type: tutorial
+title: Scrivi la tua prima riga di Astro
+description: |-
+  Tutorial: Costruisci il tuo primo blog Astro —
+  Fai le tue prime modifiche alla home page del tuo progetto tutorial
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+
+<PreCheck>
+  - Fai le tue prime modifiche al tuo nuovo sito
+</PreCheck>
+
+## Modifica la tua home page
+
+1. Nel tuo editor di codice, naviga nella sezione Explorer a `src/pages/index.astro` e cliccaci sopra per aprire il contenuto del file in una scheda modificabile.
```

Comment:
The proposition "a" doesn't sound right to me. I would suggest reassembling the phrase, for example, "apri il file `src/pages/index.astro` dalla sezione Explorer, e cliccaci sopra per aprire il suo contentuo in una scheda modificabile"